static/js/datatable_loader.js

```diff
@@ -25,14 +27,17 @@ function dataTableLoader(path) {
   }
 
   const doFetch = (tableData, callback) => {
-    fetch(`${path}?${new URLSearchParams(tableData)}`)
+    fetch(`${path}?${new URLSearchParams(tableData)}&${param}`)
```

Comment:
I am not the reviewer of this PR but I have reviewed https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2566 just now and it touches the same line of code slightly differently.

@Eisei24 , is your change here to fix the filter? It was not working on the `/slots` page before and a change in this line fixed it.

If so, please take a look at https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2566 and maybe discuss with @remoterami what approach is better.

Reply:
We want both changes, they don't really conflict with each other.
I merged the current master for this branch.